dlib/dnn/loss.h

```diff
@@ -479,9 +507,6 @@ namespace dlib
             overlaps_nms = test_box_overlap(iou_thresh, percent_covered_thresh);
         }
 
-
-    private:
```

Comment:
Why is this private removed?

Reply:
It isn't. I wanted to extract the `set_overlap_nms` function, rather than leave identical code in the two constructors. I made this new function private, so the diff just looks a little confusing here. 

In other words, it's still there – [right here](https://github.com/davisking/dlib/pull/809/files/90da2bbc0b2f42a4b0420c6cefd34437e209b165#diff-32a043d312608d62a296b05ce1bc096eR480), to be specific. :)

On the plus side, the diff doesn't show the function code itself as being deleted from one place and added into another place.

Reply:
Ah, right, looks good :)

I wish there was a github button that uncollapsed all the folds without needing to click on the expand button a billion times :/